controller/controller.go

```diff
@@ -87,6 +107,14 @@ type ObjectStorageController struct {
 	kubeClient   kubeclientset.Interface
 }
 
+func NewDefaultObjectStorageController(identity string, leaderLockName string, threads int) (*ObjectStorageController, error) {
+	rateLimit := workqueue.NewMaxOfRateLimiter(
+		workqueue.NewItemExponentialFailureRateLimiter(100*time.Millisecond, 600*time.Second),
```

Comment:
Added a default that seemed more reasonable. With the previous value, it was retrying too quickly

Reply:
This can't go here.  It has to be passed in so creators of the controller can set their backoff period.  The sidecar will have a different backoff process than everything else.